Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:latest
+
+WORKDIR /usr/src/app
+
+COPY package*.json ./
```

Comment:
This is copying just package.json, while on line 9 you are copying everything in the directory. We probably don't needs this line? Or did we have something that was happening when you removed this?